docs/MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+module(
```

Comment:
With this becoming a module I think we should also add a corresponding WORKSPACE for the legacy setup.